cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -756,7 +799,7 @@ func (csr *ClusterStateRegistry) GetStatus(now time.Time) *api.ClusterAutoscaler
 
 		// Health.
 		nodeGroupStatus.Conditions = append(nodeGroupStatus.Conditions, buildHealthStatusNodeGroup(
-			csr.IsNodeGroupHealthy(nodeGroup.Id()), readiness, acceptable, nodeGroup.MinSize(), nodeGroup.MaxSize()))
+			csr.GetNodeGroupHealth(nodeGroup.Id()).IsHealthy, readiness, acceptable, nodeGroup.MinSize(), nodeGroup.MaxSize()))
```

Comment:
nit: Why not pass the whole Health object here like we do for Safety and `buildScaleUpStatusNodeGroup`?

Reply:
It is not currently needed, but I've added it in case we need to add more error info to Health status. Thank you!